src/db/effect_bundles_tables_vco.tsv

```diff
@@ -29,3 +29,6 @@ vco_victory_payload_ogr_gre_1_rich_walk	Hoardmaster	Greasus' wealth is now so ob
 vco_victory_payload_ogr_gre_2_overtyrants_reign_1	Hell on Wheels	In recognition of Greasus' indisputable dominion over these lands, the Black Dwarfs of Zhar-Naggrund have gifted him with a powerful daemon engine to strap to the back of his cart. In return, they merely ask that he look the other way when the occasional Cathayan caravan goes missing...	battle_movement_character.png	faction	1	true	false	true
 vco_victory_payload_ogr_gre_2_overtyrants_reign_2	Blood-forged Gut-plate	Grimgor's armour was forged in the blood of Dwarf Runepriests, thus granting it potent defensive properties. As it was far too small for Greasus' magnificent bulk, it was melted down and reforged into a gut-plate of terrific proportions.	armour_character.png	faction	1	true	false	true
 vco_victory_payload_ogr_gre_3_drakecrush	Subservient Cathayans	The Cathayans have now accepted their place at the Overtyrant's feet. Caravans journey to his Great Hall laden with gold, gemstones and exotic meats, hoping to barter for safe passage across the Ivory Road. Each season, their profit margins grow smaller.	diplomacy_trespass.png	faction	1	true	false	true
+vco_victory_payload_skv_mdr_1_yes_yes_prepare_for_war_1	War Footing	Throt's mutant armies are ready and the Vermintide awaits. Hell Pit has geared up for war!	growth_juice.png	faction	1	true	false	true
```

Comment:
Why the `_1`? I can understand it when you get two or more effect bundles for the same route, but I do not think it should be there for the others. At least to keep it coherent with the already existent ones.